exonum-java-binding/cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/transactions/CreateWalletTxIntegrationTest.java

```diff
@@ -94,16 +99,17 @@ void executeCreateWalletTx(TestKit testKit) {
 
   @Test
   @RequiresNativeLibrary
+  @Disabled("Disabled until transaction results fix")
```

Comment:
add ticket number if any

Reply:
Transaction result is already in (#1174)

Reply:
Enabled these